domains/amvtesting.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "AMV0027"
+    },
+    "record": {
+        "RECORDTYPE": "RECORDCONTENT"
```

```suggestion
        "RECORDTYPE": "RECORDCONTENT"
```
RECORDTYPE is not a valid record type.